### ISSUE TITLE
Sensu client should provide an HTTP input on 3031

### ISSUE THREAD

**Issue by portertech** [sensu/sensu#1432]
The Sensu client should provide an HTTP input on port 3031. The input should essentially be an HTTP API, beginning with endpoints that implement the functionality of the current client TCP socket, e.g. accept external check results. The Sensu client HTTP input should use `em-http-server`, the same library used by the Sensu API.

This proposal is an alternative to https://github.com/sensu/sensu/pull/1352
## Expected Behavior

HTTP 3031 `POST /results`
HTTP 3031 `GET /info`

Future endpoints:

HTTP 3031 BASIC AUTH `GET /settings`
## Current Behavior

The Sensu client currently provides a TCP socket on port 3030, responding to "ping", and accepting JSON check results.

Examples:

```
echo ping | nc localhost 3030
pong
```

```
echo '{"name": "test", "output": "WARNING", "status": 1}' | nc localhost 3030
ok
```

**Comment by moises-silva** [comment]
I've taken a stab at implementing this on PR #1488

**Comment by portertech** [comment]
Merged into release/0.27
